src/bin/accuracy_reports.rs

```diff
@@ -837,7 +853,8 @@ fn map_whatlang_to_lingua(language: Option<WhatlangLanguage>) -> Option<Language
         Some(WhatlangLanguage::Mar) => Some(Language::Marathi),
         Some(WhatlangLanguage::Mkd) => Some(Language::Macedonian),
         Some(WhatlangLanguage::Nld) => Some(Language::Dutch),
-        Some(WhatlangLanguage::Nob) => Some(Language::Bokmal),
+        // Some(WhatlangLanguage::Nob) => Some(Language::Bokmal),
+        Some(WhatlangLanguage::Nob) => Some(Language::Norwegian),
```

Comment:
There is no entry for Norwegian in the `Language` enum anymore. So this will probably not compile at all. Please remove this line again and replace it with Bokmal.

Reply:
Hello again, I've made some changes according to your review a while ago, was not sure if you'd received the notification